docker/debian-latest-amd64-openssl-3.0.x/README.md

```diff
@@ -0,0 +1,7 @@
+# OS support: debian-latest-amd64-openssl-3.0.x
+
+- Base Docker image: `node:lts-slim`
+- OS: Linux Debian (latest)
+- Arch: amd64 (x86_64)
+- OpenSSL location: `/usr/lib/x86_64-linux-gnu/libssl.so.3`
```

Comment:
```suggestion
- OpenSSL location (after manual install): `/usr/lib/x86_64-linux-gnu/libssl.so.3`
```
I think we should mention that the OpenSSL was installed additionally in the READMEs somehow

Reply:
Good point, we usually added a line to the end of the README. i'm pushing another commit